sdk/scala-sdk-protobuf/src/main/scala/kalix/scalasdk/impl/InternalContext.scala

```diff
@@ -16,10 +16,16 @@
 
 package kalix.scalasdk.impl
 
+import kalix.javasdk.Metadata
+import kalix.javasdk.impl.MetadataImpl
+
 /**
  * INTERNAL API
  */
 // note cannot be package private since used by generated code
 trait InternalContext {
   def getComponentGrpcClient[T](serviceClass: Class[T]): T
+
+  /** Meant to be used by component calls, initially such they have access to info like trace parent */
```

Comment:
Please review this comment.

Reply:
As mentioned above, say it is `Metadatimpl` directly instead. Maybe something hinting that it is per call metadata would be good for the name `componentCallMetadata`?